ferrocene/ci/scripts/emulated-aarch64-qnx-test-runner.sh

```diff
@@ -113,7 +113,10 @@ cmd_prepare() {
 
     echo
     echo "===> building remote-test-server"
-    stage="${REMOTE_TEST_SERVER_STAGE-0}"
+    stage="${REMOTE_TEST_SERVER_STAGE}"
```

Comment:
According to https://stackoverflow.com/a/16753536, I think you can make this

```
stage="${REMOTE_TEST_SERVER_STAGE-1}"
```
And get the same result?